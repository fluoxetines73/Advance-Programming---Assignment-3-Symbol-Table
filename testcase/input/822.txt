RPRINT
INSERT x string
LOOKUP x
LOOKUP x
ASSIGN x '1234ABC'
ASSIGN x '1234ABC'
ASSIGN x 'Test123' random
INSERT a string
BEGIN
ASSIGN a 'Test123'
END
LOOKUP x
ASSIGN x 'HelloWorld'
INSERT y string
RPRINT
END
PRINT
INSERT x number
BEGIN
RPRINT
PRINT
BEGIN
END