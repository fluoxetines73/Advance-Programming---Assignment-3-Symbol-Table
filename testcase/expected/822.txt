Invalid: ASSIGN x 'Test123' random